src/style.css

```diff
@@ -1 +1,8 @@
-/* styles go here */
+.form_field {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.field {
+  margin-bottom: 20px;
+}
```

Comment:
you can use :not(:last) child to add margin to the all fields besides last one

Reply:
done